src/Storage/Commands.Storage/packages.config

```diff
@@ -1,10 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Storage.DataMovement" version="0.6.1" targetFramework="net452" />
+  <package id="Microsoft.Azure.Storage.DataMovement" version="0.6.3" targetFramework="net452" />
   <package id="Microsoft.Data.Edm" version="5.8.2" targetFramework="net452" />
   <package id="Microsoft.Data.OData" version="5.8.2" targetFramework="net452" />
   <package id="Microsoft.Data.Services.Client" version="5.8.2" targetFramework="net452" />
+  <package id="Microsoft.WindowsAzure.ConfigurationManager" version="1.8.0.0" targetFramework="net452" />
+  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net452" />
```

Comment:
@blueww similar comment to the .csproj file: remove this reference to `Newtonsoft.Json` as it can be found in the `packages.config` file of the `Commands.ResourceManager.Common` project.

Reply:
removed